coderd/coderd.go

```diff
@@ -991,6 +991,7 @@ func New(options *Options) *API {
 				r.Use(httpmw.RateLimit(options.LoginRateLimit, time.Minute))
 				r.Post("/login", api.postLogin)
 				r.Post("/otp/request", api.postRequestOneTimePasscode)
+				r.Post("/validate-password", api.validateUserPassword)
```

Comment:
suggestion: add a rate limit here, could also be a follow-up until we know a reasonable value

Reply:
Yeah good one - let me add a follow-up for this one 🙏